opencti-platform/opencti-front/src/private/components/arsenal/channels/ChannelCreation.tsx

```diff
@@ -156,90 +158,114 @@ export const ChannelCreationForm: FunctionComponent<ChannelFormProps> = ({
       onReset={onReset}
     >
       {({ submitForm, handleReset, isSubmitting, setFieldValue, values }) => (
-        <Form style={{ margin: '20px 0 20px 0' }}>
-          <Field
-            component={TextField}
-            name="name"
-            label={t_i18n('Name')}
-            fullWidth={true}
-            detectDuplicate={['Channel', 'Malware']}
-          />
-          <OpenVocabField
-            type="channel_types_ov"
-            name="channel_types"
-            label={t_i18n('Channel type')}
-            multiple
-            containerStyle={fieldSpacingContainerStyle}
-            onChange={setFieldValue}
-          />
-          <Field
-            component={MarkdownField}
-            name="description"
-            label={t_i18n('Description')}
-            fullWidth={true}
-            multiline={true}
-            rows="4"
-            style={{ marginTop: 20 }}
-          />
-          <ConfidenceField
-            entityType="Channel"
-            containerStyle={fieldSpacingContainerStyle}
-          />
-          <CreatedByField
-            name="createdBy"
-            style={{
-              marginTop: 20,
-              width: '100%',
-            }}
-            setFieldValue={setFieldValue}
-          />
-          <ObjectLabelField
-            name="objectLabel"
-            style={{
-              marginTop: 20,
-              width: '100%',
+        <>
+          {isFeatureEnable('BULK_ENTITIES') && (
+            <BulkTextModal
+              open={bulkModalOpen}
+              onClose={onBulkModalClose}
+              onValidate={((val) => {
+                setFieldValue('name', val);
+                if (splitMultilines(val).length > 1) {
+                  setFieldValue('file', null);
+                }
+              })}
+              formValue={values.name}
+            />
+          )}
+          <Form style={{ margin: '20px 0 20px 0' }}>
+            <Field
+              component={isFeatureEnable('BULK_ENTITIES') ? BulkTextField : TextField}
+              name="name"
+              label={t_i18n('Name')}
+              fullWidth={true}
+              detectDuplicate={['Channel', 'Malware']}
```

Comment:
why do we check duplicates on malware for channel ?

Reply:
I don't know, it was here before my work, and it is also the case for other threats entities